metricbeat/tests/system/test_jolokia.py

```diff
@@ -50,5 +50,4 @@ def test_jmx(self, mbean):
         assert evt["jolokia"]["test"]["gc"]["collection_count"] >= 0
 
     def get_hosts(self):
-        return [os.getenv('JOLOKIA_HOST', 'localhost') + ':' +
-                os.getenv('JOLOKIA_PORT', '8778')]
+        return [self.compose_host()]
```

Comment:
I see this pattern a lot, you could also replace `get_hosts()` call with `self.compose_host()` and remove `get_hosts` all together??

Reply:
Good suggestion, thanks! It should be possible, I will do it and override only when needed.